en/docs/tutorials/create-rest-api-component.md

```diff
@@ -0,0 +1,299 @@
+# Create a REST API
+
+A REST API is an Application Programming Interface that conforms to the constraints of REST architectural styles and principles. REST is an [architectural style](https://www.ics.uci.edu/~fielding/pubs/dissertation/rest_arch_style.htm) introduced by Dr. Roy Fielding in his doctoral dissertation in the year 2000. It was initially designed as a guide for the development of the web. Today, REST is being widely adopted for the creation of high-performance, stateless and reliable APIs that are used by numerous applications.
+
+Choreo’s low-code editor allows developers to easily design (and then implement) high-quality REST APIs. To explore this capability, let's consider a scenario where an Analyst needs to retrieve the daily count of COVID-19 patients per one million population by country. In this tutorial, you will create an API that supports this.
+
+## Develop
+
+In this section, let's develop the application that retrieves COVID-19 related statistics.
+
+### Create a project and add a REST API component
+
+1. Sign in to the Choreo Console at https://consolev2.preview-dv.choreo.dev/.
+
+2. Expand the drop-down menu for projects and click **Create Project**.
+
+    ![Create project](../assets/img/tutorials/rest-api/create-project.png){.cInlineImage-full}
+
+3. Enter the name and the description for the project as follows:
+
+    | **Field**       | **Value**                             |
+    |-----------------|---------------------------------------|
+    | **Name**        | `COVID-19Stats`                       |
+    | **Description** | `Get statistics relating to COVID-19` |
+
+4. Click **Create**.
+
+### Add a REST API component
```

Comment:
Numbered it. Did not use the word "step" because this is a substep, and avoided using the word "substep" also because it did not seem nice using that word in a title. What do you think?